administrator-guides/custom-sounds/README.md

```diff
@@ -0,0 +1,27 @@
+# Custom Sounds
+
+Rocket.Chat allows you to add cutom sound that can be used as notification sound by other users for different channels.
+
+You can add your own mp3 files.
+
+## Add a custom sound
+
+- Go to the Administration tab
+- Select the Custom Sounds
+- Click on the Add symbol (+) in the right side bar
+- Enter a name for the sound
+- Browse and choose a mp3 file
+- Press the SAVE button
+
+## Using custom sound
+
+- To use the added sound as a notification for an channel select `Notifications Preferences` from the `More` section of the channel.
```

Comment:
Ideally, we should have this part on a dedicated `notifications preferences` article under the user guides and only link to said article. We should not "hide" user guides under an admin article

Reply:
Yes, I thought to provide a link to the `Notification Preferences` here. But I was unable to find any `Notification Preference` article or any suitable heading under which it can come. So should I add `Notification Preference` somewhere and update this PR with the link?